### PR TITLE
PUB-493

### PR DESCRIPTION
**Pull request by ejgullo**
- Adding (mostly) empty components for Search Results page
- Adding reactable package
- Adding some styling
- Adding footer
- Adding Routes

### CODE REVIEW

File: src/js/components/Footer.jsx [hunk 1]
```diff
@@ -0,0 +1,18 @@
+/**
+ * FooterComponent.jsx
+ * Created by Mike Bray 12/26/15
```
Comment (mab6bf) [inline src/js/components/Footer.jsx]: Might want to update this line

File: src/js/components/search/SearchSidebar.jsx [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+import React from 'react';
+import SearchOption from './SearchOption.jsx';
+
+export default class SearchSidebar extends React.Component {
+    constructor(props) {
+        super(props);
+    }
+
+    render() {
+        let options = [
```
Comment (mab6bf) [inline src/js/components/search/SearchSidebar.jsx]: Bring this out of render() - if it's static, we don't need to reinitialize during every render

File: src/js/containers/BarChart.jsx [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+		super(props);
+	}
+
+
+  render() {
+
+    var BarChart = rd3.BarChart;
+
+    // placeholder info, pass in info from API
+    var barData = [
```
Comment (mab6bf) [inline src/js/containers/BarChart.jsx]: Same thing as above - we can move this outside of the render function

File: src/js/containers/BarChart.jsx [hunk 4]
```diff
@@ -0,0 +1,57 @@
+/**
+ * BarChart.jsx
+ * Created by Emily Gullo 10/14/2016
+ **/
+import rd3 from 'rd3';
+import React from 'react';
+// import { kGlobalConstants } from '../GlobalConstants.js';
```
Comment (mab6bf) [inline src/js/containers/BarChart.jsx]: Should this be included? If not, let's remove

File: src/js/containers/Map.jsx [hunk 5]
```diff
@@ -0,0 +1,19 @@
+/**
+ * Map.jsx
+ * Created by Emily Gullo 10/14/2016
+ **/
+import React from 'react';
+// import { kGlobalConstants } from '../GlobalConstants.js';
```
Comment (mab6bf) [inline src/js/containers/Map.jsx]: Should this be included? If not, let's remove

File: src/js/containers/Table.jsx [hunk 6]
```diff
@@ -0,0 +1,28 @@
+/**
+ * Table.jsx
+ * Created by Emily Gullo 10/14/2016
+ **/
+import React from 'react';
+import reactable from 'reactable';
+// import { kGlobalConstants } from '../GlobalConstants.js';
```
Comment (mab6bf) [inline src/js/containers/Table.jsx]: Should this be included? If not, let's remove

File: src/js/containers/router/RouterContainer.jsx [hunk 7]
```diff
@@ -0,0 +1,53 @@
+/**
+* RouterContainer.jsx
+* Created by Kevin Li 3/16/15
```
Comment (mab6bf) [inline src/js/containers/router/RouterContainer.jsx]: Take credit for this!

File: src/js/containers/router/RouterRoutes.jsx [hunk 8]
```diff
@@ -0,0 +1,62 @@
+/**
+ * SearchResults.jsx
+ * Created by Kevin Li 3/16/15
```
Comment (mab6bf) [inline src/js/containers/router/RouterRoutes.jsx]: Same as above

**Comment by ejgullo** [comment]
@mab6bf changes made, along with some additional scss updates

**Pull request merged**